src/components/main/Canvas.tsx

```diff
@@ -39,6 +42,8 @@ const MainContainer = styled('div')(() => ({
 const Canvas = (): JSX.Element => {
   const [backgroundToggleSetting, setBackgroundToggleSetting] = useState(false);
   const context = useContext(Context);
```

Comment:
I think you can remove this lien since you are using the `useLocalContext` hook which is the same.

Reply:
Thanks, I'll fix this.